### DISCUSSION TITLE
Datetime fields (Timezone)

### DISCUSSION THREAD

**Discussion by cole007** [craftcms/cms#12099] [Ideas] [answered]
When specifying the settings for a Datetime field it is great to provide the ability for a user/author to specify the timezone of a particular date and time.

However, what would be really helpful for a site with eg events where the author and the events are consistently in different time zones for the Field Settings to permit a default timezone as it is quite laborious to choose a particular timezone from the list on large numbers of entries.

<img width="1057" alt="Screenshot 2022-10-12 at 09 35 30" src="https://user-images.githubusercontent.com/855195/195293327-c3dad509-5448-46eb-bd87-f5b68e08ed89.png">

**Comment by brandonkelly** [comment, marked as answer]
It already defaults to the system time zone (Settings → General → Time Zone), which I imagine would generally be the desired default value?

> as it is quite laborious to choose a particular timezone from the list on large numbers of entries.

Worth noting that the time zone selection UI has improved in Craft 4, with better keyword searching:

<img width="405" alt="the time zone selector for a Date field in Craft 4, with an active search for “lon”, showing four results starting with London." src="https://user-images.githubusercontent.com/47792/196277766-27c975c2-122e-4706-8fa2-00fd9b8c222e.png">
